### DISCUSSION TITLE
Change the discovery server id on the client create3 platform.

### DISCUSSION THREAD

**Discussion by y2d2** [iRobotEducation/create3_docs#431] [Questions] [answered]
I am not sure if it is possible already today, but I was wondering if it was possible to change the discovery server id for the client create3 platform. It seems today it is assumed always to be 0 id (Corresponding to the `fastdds discovery -i 0 -p 11811` command on the server PC, as specified by the documentation https://iroboteducation.github.io/create3_docs/setup/discovery-server/).
 However, in a setup with multiple platforms, it could be beneficial to have them on different servers. 
![image](https://github.com/iRobotEducation/create3_docs/assets/10230591/b21e6439-e3c2-424b-b820-f4b701bf9cb8)

**Comment by shamlian** [comment, marked as answer]
I believe that you can specify multiple server IDs using `;` in the "address and port of Fast DDS discovery server" box using the same syntax as for the `ROS_DISCOVERY_SERVER` environment variable described in the [Server Redundancy](https://docs.ros.org/en/humble/Tutorials/Advanced/Discovery-Server/Discovery-Server.html#server-redundancy) section of the ROS 2 docs. The support to allow this `;` character in the box was added in H.2/G.5 and greater versions. See also #400.

**Comment by y2d2** [comment]
Hi Shamlian! 

Thank you, for replying so fast. For some reason, I thought it would not work like that. 

Cheers, 

Yuri
